cmd/wego/flux/cmd.go

```diff
@@ -11,10 +11,11 @@ import (
 )
 
 var Cmd = &cobra.Command{
-	Use:     "flux -- [flux commands or flags]",
-	Short:   "Use flux commands",
-	Example: "wego flux -- install -h",
-	Run:     runCmd,
+	Use:                "flux [flux commands or flags]",
+	Short:              "Use flux commands",
+	DisableFlagParsing: true,
```

Comment:
Nice finding

Reply:
hahah yep, was looking for something else and I looked to that and had aha 💡 moment.